modules/cluster.py

```diff
@@ -195,11 +200,13 @@ def get_best_cluster_block_align(read_cl_id, representatives, hit_clusters_ids,
             gap_opening_penalty = 2
 
         match_id_tailored = math.floor((1.0 - error_rate_sum) * args.k)
-        (s1, s2, (s1_alignment, s2_alignment, alignment_ratio)) = parasail_block_alignment(seq, c_seq, args.k, match_id_tailored, opening_penalty = gap_opening_penalty,  )
+        (s1, s2, (s1_alignment, s2_alignment, alignment_ratio, target_alignment_ratio)) = parasail_block_alignment(seq, c_seq, args.k, match_id_tailored, opening_penalty = gap_opening_penalty,  )
         # print("Expected errors:", poisson_mean, poisson_mean2)
-        if alignment_ratio >= args.aligned_threshold: #args.mapped_threshold:
+        if args.symmetric_map_align_thresholds and min(alignment_ratio, target_alignment_ratio) >= args.aligned_threshold:
```

Comment:
same here, wouldn't it be desirable to return `min(alignment_ratio, target_alignment_ratio)`

Reply:
Thank you, updating PR.